x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/activity_monitor/columns.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiIconTip, EuiLink, EuiTextColor } from '@elastic/eui';
+import React from 'react';
+import { getEmptyTagValue } from '../../../../components/empty_value';
+import { ColumnTypes } from './index';
+
+const actions = [
+  {
+    available: (item: ColumnTypes) => item.status === 'Running',
+    description: 'Stop',
+    icon: 'stop',
+    isPrimary: true,
+    name: 'Stop',
+    onClick: () => {},
+    type: 'icon',
+  },
+  {
+    available: (item: ColumnTypes) => item.status === 'Stopped',
+    description: 'Resume',
+    icon: 'play',
+    isPrimary: true,
+    name: 'Resume',
+    onClick: () => {},
+    type: 'icon',
+  },
+];
+
+// Michael: Are we able to do custom, in-table-header filters, as shown in my wireframes?
+export const columns = [
+  {
+    field: 'rule',
+    name: 'Rule',
+    render: (value: ColumnTypes['rule']) => <EuiLink href={value.href}>{value.name}</EuiLink>,
+    sortable: true,
+    truncateText: true,
+  },
+  {
+    field: 'ran',
+    name: 'Ran',
+    render: (value: ColumnTypes['ran']) => '--',
```

Comment:
Shouldn't all of these be the `getEmptyValue()` from:
```ts
x-pack/legacy/plugins/siem/public/components/empty_value/index.tsx
```

Since we prefer single dash for empty values?

Reply:
Everything under `activity_monitor/` is all initial placeholder content -- I just moved it into its own component. But yeah, when/if that table is implemented we'll want to use `getEmptyValue()`.